components/form/form.js

```diff
@@ -139,23 +139,25 @@ class Form extends FormMixin(LitElement) {
 		}
 		e.stopPropagation();
 		e.preventDefault();
+
 		const form = e.composedPath()[0];
+		const target = e.target;
```

Comment:
The only change here is the pull out a copy of `e.target` into a local variable, instead of referring to it as `e.target` everywhere. Why? The answer is interesting.

Normally it wouldn't matter, but as the original event bubbles up through the DOM, as soon as it crosses a shadow root boundary, the target changes to the new element to "hide" its internals. So after this initial code runs, `e.target` can be an element further up the DOM, which means looking for it later in the `this._nestedForms` Map isn't going to find anything.

With my original test code this wasn't an issue, since there was only 1 level of custom elements. But the Creator case had multiple layers. I've added a new test case to handle this.

Reply:
TIL. I think that explain som weird behaviour I've seen before.